packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -71,57 +69,65 @@ export const PaymentConfirmation = ({
   const downloadUrl: URL = new URL(productUrl);
   downloadUrl.searchParams.append('email', email);
 
+  // TW classes
+  const subheadingClasses = 'text-grey-400 max-w-sm';
+  const h2classes = 'text-xl font-normal mx-0 mt-6 mb-3';
+  const h3classes = 'text-sm font-semibold';
+  const bottomRowClasses = 'flex justify-between items-center text-grey-400';
+
   return (
     <>
       <SubscriptionTitle screenType="success" />
       <section
-        className={`container card payment-confirmation ${className}`}
+        className="container payment-confirmation"
         data-testid="payment-confirmation"
       >
-        <header>
+        <header
+          className={`flex flex-col justify-center items-center row-divider-grey-200 text-center pb-8 mt-5 desktop:mt-2`}
+        >
           {accountExists ? (
             <>
               <img
-                className="circled-check"
+                className="max-h-12"
                 src={circledCheckbox}
                 alt="circled checkbox"
               />
               <Localized id="payment-confirmation-thanks-heading">
-                <h2>Thank you!</h2>
+                <h2 className={h2classes}>Thank you!</h2>
```

Comment:
I'm not 100% how l10n stuff works. By adding a `className` do need to change the localized id?
Or maybe not, since it looks like [`payment-confirmation-thanks-heading`](https://github.com/mozilla/fxa/blob/973b1685d063e7e1321896686cb2181470e9c892/packages/fxa-payments-server/public/locales/en-US/main.ftl#L405) only contains the text, and not the markup?

Reply:
I don't believe it needs to be updated either, but just in case - @mozilla/fxa-l10n, can you please confirm? Currently in `main.ftl` it's: 
`payment-confirmation-thanks-heading = Thank you!`

Reply:
@xlisachan / @pdehaan  - Confirming that adding a className would not affect the text here and no new ID is required.

Reply:
Thanks all!